test/file_io_test.py

```diff
@@ -132,9 +132,16 @@ async def container_filesystem_exec_get_output(servicer, stream):
         f.close()
 
 
-def test_file_write_large(servicer, client):
+@pytest.mark.parametrize(
+    "file_size, expected_counter",
+    [
+        (int(5.5 * WRITE_CHUNK_SIZE), 6),
+        (int(2 * WRITE_CHUNK_SIZE), 2),
+    ],
+)
+def test_file_write_chunks(servicer, client, file_size, expected_counter):
     """Test file write chunking logic."""
-    content = "A" * WRITE_FILE_SIZE_LIMIT
```

Comment:
Do you think it was important to be testing that the write at the declared limit?

Reply:
I do not think it's important to test up to the limit of 1 GiB. The slowness of this test came from chunking up to this limit.

For this test, I think it's more important to make sure the chunking worked correctly for different sizes. (Which was added in this PR)